source/features/git-checkout-pr.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import ClippyIcon from 'octicon/clippy.svg';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import {getCurrentBranch, getPRRepositoryInfo} from '../github-helpers';
+
+const connectionType: Record<string, string> = {
+	HTTPS: `${location.origin}/`,
+	SSH: `git@${location.hostname}:`
+};
+
+function checkoutOption(option: string): JSX.Element {
+	const {owner: user, name: repository} = getPRRepositoryInfo();
```

Comment:
Is this malicious compliance? Just use `owner` and `name`

Reply:
No I did not want to go and change it everywhere. I think I ask going to revert and put it in a pint.